rocketpy/simulation/flight.py

```diff
@@ -3260,7 +3385,31 @@ class attributes which are instances of the Function class. Usage
             encoding="utf-8",
         )
 
-        return
+    def export_sensor_data(self, file_name, sensor=None):
+        """Exports sensors data to a file. The file format can be either .csv or
+        .json.
+
+        Parameters
+        ----------
+        file_name : str
+            The file name or path of the exported file. Example: flight_data.csv
+            Do not use forbidden characters, such as / in Linux/Unix and
+            `<, >, :, ", /, \\, | ?, *` in Windows.
+        sensor : Sensor, optional
+            The sensor to export data. If None, all sensors data will be exported.
+            Default is None.
+        """
+        if sensor is None:
+            data_dict = {}
+            for key, value in self.sensor_data.items():
+                data_dict[key.name] = value
```

Comment:
What is key and value? I know that key is a sensor, but what is value?

Reply:
https://github.com/RocketPy-Team/RocketPy/pull/583/commits/75f8d9e47900ba2798661d88fb279bd44704f0cd